### PR TITLE
Chore/replace web dev server with vite

### PR DESCRIPTION
**Pull request by Sarin-Udompanish**
## Description
Replacing `web-dev-server` in all packages with vite and remove theme loader from `demo-block` element.

## Type of change
- [x] Refactor (improves code without changing its functionality)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -13,6 +13,12 @@
...
     "*.config.js",
     "__test__",
-    "*.md"
+    "*.md",
+    "cli.js",
+    "**/lib",
+    "__demo__",
+    "__snapshots__",
+    "scripts",
+    "samples"
```
Comment (wsuwt) [inline .eslintrc:22]: Should this be added to v6 as well ?

File: packages/demo-block/src/demo-block.ts [hunk 2]
```diff
@@ -30,32 +33,19 @@ const getPrevTheme = (): string => {
   return Themes[(Themes.indexOf(getCurrentTheme()) + Themes.length - 1) % Themes.length];
 };
 
-/* c8 ignore start */
+/* istanbul ignore next */
```
Comment (wsuwt) [inline packages/demo-block/src/demo-block.ts]: This should be `c8` in v7

File: packages/elements/src/button/__demo__/index.html [hunk 3]
```diff
@@ -37,14 +36,31 @@
...
+
+      import (`../../../../../node_modules/@refinitiv-ui/${theme}-theme/${variant}/css/native-elements.css`);
+      import (`../../../../../node_modules/@refinitiv-ui/${theme}-theme/${variant}/ef-button.js`);
+    </script>
+ 
     <!-- DEFAULT -->
     <demo-block layout="normal" header="Default" tags="default, disabled">
       <ul class="demos">
         <li>
+          <button>sadsadsad</button>
```
Comment (wsuwt) [inline packages/elements/src/button/__demo__/index.html]:
```suggestion
```

File: packages/elements/src/heatmap/__demo__/event.html [hunk 4]
```diff
@@ -4,7 +4,6 @@
 <head>
   <meta charset="utf-8">
   <title>ef-heatmap</title>
-  <link rel="stylesheet" href="/node_modules/@refinitiv-ui/demo-block/demo.css">
```
Comment (wsuwt) [inline packages/elements/src/heatmap/__demo__/event.html:7]: Missing theme import.

File: packages/i18n/__demo__/index.html [hunk 5]
```diff
@@ -4,9 +4,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>i18n Translate Benchmark</title>
-    <link rel="stylesheet" href="/node_modules/@refinitiv-ui/demo-block/demo.css">
   </head>
   <style>
+    body {
+      visibility: visible !important;
+    }
```
Comment (wsuwt) [inline packages/i18n/__demo__/index.html]: Why do we need this ?

**Review by wsuwt — APPROVED** [review]

**Review by wsuwt — APPROVED** [review]

**Review by wsuwt — APPROVED** [review]

File: packages/i18n/__demo__/index.html (hunk 5 above)
Comment (Sarin-Udompanish) [reply packages/i18n/__demo__/index.html]: Theme loader in demo block will add this style to body tag when theme has loaded but we don't have theme loader anymore.

File: .eslintrc (hunk 1 above)
Comment (Sarin-Udompanish) [reply .eslintrc:22]: We have rejected this pattern once in v6 https://github.com/Refinitiv/refinitiv-ui/pull/475.

**Review by wsuwt — APPROVED** [review]

**Pull request merged**